source/imaer-gml/src/main/java/nl/overheid/aerius/gml/GMLValidator.java

```diff
@@ -47,6 +49,13 @@ public static void validateYear(final int year, final List<AeriusException> exce
     }
   }
 
+  public static void validateSituationType(final SituationType type, final ImportParcel parcel) {
+    if(type == SituationType.NETTING) {
+      parcel.getWarnings().add(new AeriusException(ImaerExceptionReason.GML_DEPRECATED_NETTING_UPDATED_TO_OFF_SITE_REDUCTION,
+          "Deprecated situation type NETTING will be updated to OFF_SITE_REDUCTION."));
+    }
+  }
```

Comment:
I think we should just convert `NETTING` to `OFF_SITE_REDUCTION` on import, and make sure that we don't return any GMLs containing `NETTING` anymore. For old GMLs we'll always have to support a `NETTING` value anyway.

When we actually remove the deprecated `NETTING` type in IMAER we probably need a separate enum containing removed types for older GML versions. Perhaps we could introduce that one now, and only use `OFF_SITE_REDUCTION` in our domain objects?

(also, missing a space after the `if`)

Reply:
I've interpreted your comment as introducing a new enum for all situationtypes including netting, removing `NETTING` from `SituationType`, and mapping the `LegacySituationType` to `SituationType` in `GMLReader`. (see my last commit). If you had something else in mind, please let me know because I'm not sure this is much better :sweat_smile:

Reply:
Yes, I think it's better, now there's no chance we'll be using `NETTING` anywhere in non-GML code.